corehq/apps/ota/utils.py

```diff
@@ -212,8 +201,9 @@ def _inner(request, domain, *args, **kwargs):
 def update_device_id(user, device_id):
     if device_id and isinstance(user, CommCareUser):
         if not user.is_demo_user:
+            # this only updates once per day for each device
             updated = user.update_device_id_last_used(device_id)
+            if toggles.ENIKSHAY.enabled(user.domain):
+                updated = set_enikshay_device_id(user, device_id) or updated
```

Comment:
@snopoke the eNikshay custom stuff is linked in here now, which makes the dependency very explicit (it used to happen in a signal).  We've identified another approach which moves this stuff out of the restore code more into custom land (a custom fixture generator).  If having this here is a headache for ICDS related optimizations, we can bump the priority of that change and drop this.  Lemme know if that's the case.